ivy/functional/frontends/torch/tensor.py

```diff
@@ -1698,6 +1698,25 @@ def floor_(self):
         self.ivy_array = self.floor().ivy_array
         return self
 
+    @numpy_to_torch_style_args
+    @with_unsupported_dtypes(
+        {
+            "2.0.1 and below": (
+                "int8",
+                "int16",
+                "uint8",
+                "uint16",
+                "uint32",
+                "uint64",
+                "bfloat16",
+                "float64",
```

Comment:
```suggestion
                "bfloat16",
                "complex",
                "float64",
                "int8",
                "int64",
```
Updated the unsupported dtypes